hvillacrusisCT/dotTheory---TypeScript
# JS-703: The Course Project Setup

**Related Ticket**
[JS-703](https://codeandtheory.atlassian.net/browse/JS-703)

**Description**
Getting Started with Typescript - Understanding Typescript

**Validation**
1. Check the root folder to ensure that the following files exist:
   - index.html
   - package.json
   - package-lock.json
   - app.ts
   - app.js
2. Verify that `lite-server` is included in the `devDependencies` section of the `package.json` file.
3. Run `npm install`.
4. Run `npm start`.
5. Confirm that the lite server opens the `index.html` file at `localhost:3000`.
6. Make any updates to the `app.ts` file and ensure there are no errors.
7. Run `tsc app.ts`.
8. Verify that the `app.js` file is updated based on your changes.
9. The live server should automatically update as you save your changes.
